Configuration/TypoScript/Elements/010_Extensions/050_News.t3s

```diff
@@ -21,7 +21,7 @@ plugin.tx_news {
                 title = Library AAC
                 description = RSS Feed Library AAC
                 link = {$plugin.tx_news.rss.channel.link}
-                language = en_EN
+                language = en_gb
```

Comment:
Still wrong. Has to be "en-gb" to validate.

Reply:
... and what about definition in 001_Config.t3s? Does that has to be changed, too?